docs/source/conf.py

```diff
@@ -352,7 +352,7 @@
     ("py:class", "plotly.graph_objects.Figure"),
     ("py:class", "PIL.Image.Image"),
     ("py:class", "mlflow.deployments.base.BaseDeploymentClient"),
-    ("py:class", "mlflow.deployments.server.config.Endpoint"),
+    ("py:class", "Endpoint"),
```

Comment:
side-note: The `nitpick_ignore` list becomes very long, we may want to seek if we can make sphinx autodoc work with `TYPE_CHECKING` trick. Let me take a lazy solution now and file a JIRA for the better option.